### PR TITLE
0.3.x: Fix ui-tests and pin dependencies

### PR DESCRIPTION
**Pull request by martinRenou**

### CODE REVIEW

File: .github/workflows/ui-tests.yml [hunk 1]
```diff
@@ -16,6 +16,8 @@ jobs:
 
       - name: Base Setup
         uses: jupyterlab/maintainer-tools/.github/actions/base-setup@v1
+        with:
+          python-version: ${{ matrix.python-version }}
```
[the remark this answers is not in the document]
Comment (martinRenou) [reply .github/workflows/ui-tests.yml:20]: Oh right, so the proper fix would be to rename the `matrix.python` to `matrix.python-version` (I realize my PR is not even correctly getting the matrix value)

**Comment by jtpio** [comment]
I restarted the UI tests check several times but it seems to be failing consistently.

**Comment by jtpio** [comment]
Maybe we should pin other dependencies like bqplot as well for the visual test. JupyterLab does something like this: https://github.com/jupyterlab/jupyterlab/blob/8e08e4252a79a6c816535b9e80759adff984cad7/pyproject.toml#L89-L101

**Comment by martinRenou** [comment]
I agree we should do this 👍🏽 I will do a similar PR against `main` with widgets packages compatible with 8.

**Comment by martinRenou** [comment]
Update Galata References

**Review by jtpio — APPROVED** [review]
Thanks!

**Pull request merged**